### PR TITLE
DIV-5168: Updated routing logic to include DA 

### PR DESCRIPTION
**Pull request by johnoneill46**
https://tools.hmcts.net/jira/browse/DIV-5168

Updated routing logic to redirect users to DA depending on case state

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: infrastructure/prod.tfvars [hunk 1]
```diff
@@ -17,3 +17,4 @@ rate_limiter_enabled = true
 
 petitioner_frontend_url = "https://www.apply-divorce.service.gov.uk"
 respondent_frontend_url = "https://www.respond-divorce.service.gov.uk"
+decree_absolute_frontend_url = "https://div-da-aat.service.core-compute-aat.internal"
```
Comment (tchow8) [inline infrastructure/prod.tfvars]: Make this prod. Will be https://decree-absolute.apply-divorce.service.gov.uk

File: infrastructure/demo.tfvars [hunk 2]
```diff
@@ -14,5 +14,6 @@ rate_limiter_enabled = false
 
 petitioner_frontend_url = "https://div-pfe-demo.service.core-compute-demo.internal"
 respondent_frontend_url = "https://div-rfe-demo.service.core-compute-demo.internal"
+decree_absolute_frontend_url = "https://div-da-aat.service.core-compute-aat.internal"
```
Comment (tchow8) [inline infrastructure/demo.tfvars]: demo instead of aat

Comment (johnoneill46) [reply infrastructure/demo.tfvars]: Changes made

File: infrastructure/prod.tfvars (hunk 1 above)
Comment (johnoneill46) [reply infrastructure/prod.tfvars]: Changes made

File: infrastructure/prod.tfvars [hunk 3]
```diff
@@ -17,3 +17,4 @@ rate_limiter_enabled = true
 
 petitioner_frontend_url = "https://www.apply-divorce.service.gov.uk"
 respondent_frontend_url = "https://www.respond-divorce.service.gov.uk"
+decree_absolute_frontend_url = "https://decree-absolute.apply-divorce.service.gov.uk"
```
Comment (tchow8) [inline infrastructure/prod.tfvars]: whoops, sorry my bad. needs www. 
 I think the DNS doesn't actually handle non www. lol

**Comment by nsingamsetty** [comment]
Code Merged as the tests passed.Please refer DIV-5168 for evidences and sign off.

**Pull request merged**